conf/modules.config

```diff
@@ -237,10 +237,22 @@ process {
         ]
     }
 
+    withName: '.*:SPLITNCIGAR:GATK4_SPLITNCIGAR:GATK4_SPLITNCIGARREADS' {
+        ext.args    =  '--create-output-bam-index false'
+    }
+
+    withName: '.*:SPLITNCIGAR:GATK4_SPLITNCIGAR:SAMTOOLS_INDEX' {
+        ext.args    =  params.bam_csi_index ? '-c' : ''
+    }
+
```

Comment:
```suggestion
```

Isn't that case covered by lines 252 to 254 already?

Reply:
No. it is the indexing of the merged BAM.

Reply:
Do you think a `withName: '.*:SPLITNCIGAR*SAMTOOLS_INDEX' {` could be working?
but we might loose some clarity.
I'm fine as it is.